beelivers_auction/sources/nft.move

```diff
@@ -77,22 +76,45 @@ public fun image_url(_: &WlNFT): Url {
 // ===== Entrypoints and functions =====
 
 /// Create a new devnet_nft
-public fun mint_many(cap: &Publisher, recipients: vector<address>, ctx: &mut TxContext) {
+public fun mint_many_and_transfer(
+    cap: &Publisher,
+    recipients: vector<address>,
+    ctx: &mut TxContext,
+) {
```

Comment:
why this changed? Do you have same formatter as we have? Make sure your editorconfig or prettier are setup correctly.